qp/quant_pdf.py

```diff
@@ -53,18 +53,17 @@ def pad_quantiles(quants, locs):
         pad_hi = False
     if n_out == n_vals:
         return quants, locs
-    ratio = (locs[:, -1] - locs[:, 0]) / (quants[-1] - quants[0])
     quants_out = np.zeros((n_out), quants.dtype)
     locs_out = np.zeros((locs.shape[0], n_out), quants.dtype)
     quants_out[offset_lo:n_vals+offset_lo] = quants
     locs_out[:,offset_lo:n_vals+offset_lo] = locs
```

Comment:
@aimalz : this below is what I understand the extrapolation on the cdf function should look like, in order to pad it to 0 on the left and 1 on the right. I did not put the epsilon everywhere though, maybe we should?